drip/models.py

```diff
@@ -43,11 +46,12 @@ class Meta:
         abstract = True
 
     @property
-    def drip(self):
+    def drip(self) -> "DripBase":
```

Comment:
This can be moved to DripBase as a class method. For instance, DripBase.create_from_abstract_drip(drip: AbstractDrip) -> DripBase, so you can avoid the circular import

Reply:
I will change this in the future. Thanks!